### PR TITLE
fix(Testimonial): mobile sizing bug fix

### PR DESCRIPTION
**Pull request by cstatro**
### PR Checklist
fixes a style bug with our testimonial component on mobile viewports. 
before:
image
after:
image

view this on mobile screen-size at the link below. 
https://tengu.codecademy.com/learner-stories?PR_ENV=portal-app-pr-555

- [ ] Related to designs:
- [x] Related to JIRA ticket: DISC-197
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.56-alpha.8c23d4.0<br/>@codecademy/gamut-labs@19.23.19-alpha.8c23d4.0<br/>@codecademy/styleguide@43.11.2-alpha.8c23d4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61312cb91bfb0114d8dc9cf9--gamut-preview.netlify.app>https://61312cb91bfb0114d8dc9cf9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61312cb91bfb0114d8dc9cf9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.56-alpha.c52428.0<br/>@codecademy/gamut-labs@19.23.19-alpha.c52428.0<br/>@codecademy/styleguide@43.11.2-alpha.c52428.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://613138d9fece251fabf7862c--gamut-preview.netlify.app>https://613138d9fece251fabf7862c--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/613138d9fece251fabf7862c>Deploy Logs</a></p>

File: packages/gamut-labs/src/Testimonial/index.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const TestimonialContent = styled(Box)(
     base: {
       display: 'grid',
       color: 'text-accent',
-      gridTemplateColumns: 'max-content max-content 1fr',
+      gridTemplateColumns: 'minmax(0, max-content) minmax(0, max-content) minmax(0, 1fr);',
```
Comment (codecaaron) [inline packages/gamut-labs/src/Testimonial/index.tsx:40]: ⛳ 
```suggestion
      gridTemplateColumns: 'repeat(2, minmax(0, max-content)) minmax(0, 1fr);',
```

**Review by codecaaron — APPROVED** [review]
🚢

**Review by dreamwasp — APPROVED** [review]
✨

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.56-alpha.48f7f7.0<br/>@codecademy/gamut-labs@19.23.19-alpha.48f7f7.0<br/>@codecademy/styleguide@43.11.2-alpha.48f7f7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://613252f3bac5dd5c29af3913--gamut-preview.netlify.app>https://613252f3bac5dd5c29af3913--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/613252f3bac5dd5c29af3913>Deploy Logs</a></p>

**Review by borisonr — APPROVED** [review]
looks good!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.59-alpha.448af1.0<br/>@codecademy/gamut-labs@19.24.3-alpha.448af1.0<br/>@codecademy/styleguide@44.0.1-alpha.448af1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61325bc742ef1252ea2af328--gamut-preview.netlify.app>https://61325bc742ef1252ea2af328--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61325bc742ef1252ea2af328>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.59-alpha.0bdfd3.0<br/>@codecademy/gamut-labs@19.24.3-alpha.0bdfd3.0<br/>@codecademy/styleguide@44.0.1-alpha.0bdfd3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://61325f88dcfbb48e93e3844d--gamut-preview.netlify.app>https://61325f88dcfbb48e93e3844d--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/61325f88dcfbb48e93e3844d>Deploy Logs</a></p>

**Pull request merged**